### DISCUSSION TITLE
Keep Cameras Always Awake

### DISCUSSION THREAD

**Discussion by four43** [homebridge-simplisafe3/homebridge-simplisafe3#365] [Q&A] [answered]
Is there a way to keep the cameras awake all the time? They take so long to boot up. My cameras are all hard wired into power supplies.

**Comment by shamoon** [comment, marked as answer]
In short no, streaming the cameras messes with motion detection etc. Is it the same in the SS app or just here? The processing power of your HB device is relevant since unfortunately we do need to transcode the stream. That seems to be the biggest thing that affects time-to-stream-start

if it’s also in the SS app that’s something to troubleshoot eg Wi-Fi etc.

ps. Assuming you’re not talking about the outdoor cams (which this plug-in doesn’t support and everyone says are horribly slow)

**Reply by four43** [reply]
@shamoon thanks for the response. I have a full sized server ready to process the streams but definitely understand if it's a limitation of the hardware. The Simplisafe app is equally as bad. Ah well.

Thanks for the info, and the work on the plugin. Super easy to use.

**Reply by four43** [reply]
The doorbell is incredibly slow too, my god lol. What are these guys doin over there?
